benchexec/benchexec.py

```diff
@@ -152,6 +152,13 @@ def create_argument_parser(self):
                           metavar="N",
                           help="Limit each run of the tool to N CPU cores (-1 to disable).")
 
+        parser.add_argument("-s", "--allowedCores", dest="coreset",
+                            default=None,
+                            metavar="n",
+                            nargs='+',
```

Comment:
Instead of using `nargs`, I would suggest making it possible to give a `1-3,5` style list. We have an existing utility method `parse_int_list` for this.